src/phases/tpch/normalized/Q10.yml

```diff
@@ -0,0 +1,39 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run TPC-H query 10 against the normalized schema. Using an 'executionStats' explain causes each command to run its execution plan until no
```

Comment:
I don't know a ton about the system under test, but is the numbering system used here (e.g. "query 6") a property of the type of query being run? Or is it a numbering system used just to organize this workload?

If the latter, would it make sense to have more descriptive names of these queries that describe what they are and why they're tested?

(I might just be totally misunderstanding this query though!)

Reply:
Gotcha, thanks for explaining!